### ISSUE TITLE
Browser Extension

### ISSUE THREAD

**Issue by Bkbailey** [LN-Juggernaut/juggernaut-desktop#27]
Would it be possible to create a browser extension for juggernaut that may also connect with the desktop app? It would be really cool if you could use juggernaut to replace joule. In a lot of ways this feels reminiscent to AOL’s AIM, but the new frontier. Use juggernaut to chat with friends, to serve as your login tool for websites and pay invoices for lightning enabled websites.

**Comment by jamespeerless** [comment]
Yes, I think it will be an important component in the future for sure.

**Comment by johncantrell97** [comment]
Yes, I believe we will likely a browser extension (maybe a fork of Joule?) at some point to help implement browser specific functionality we need.
